### ISSUE TITLE
Why my Loss changed strangely

### ISSUE THREAD

**Issue by hungcuiga1** [mozilla/DeepSpeech#3584]
I train model with [tutorial ](https://deepspeech.readthedocs.io/en/latest/TRAINING.html)of DeepSpeech v0.9.3, my cmd is :
`python3 DeepSpeech.py --n_hidden 2048 --checkpoint_dir fine_tuning_checkpoints/ --epochs 1 --train_files /mnt/c/Users/tuanh/OneDrive/Desktop/train_speech/train.csv --dev_files /mnt/c/Users/tuanh/OneDrive/Desktop/train_speech/dev.csv --test_files /mnt/c/Users/tuanh/OneDrive/Desktop/train_speech/test.csv --learning_rate 0.0001 --export_dir output_models/ --automatic_mixed_precision`
and receive result loss is:
![image](https://user-images.githubusercontent.com/44896903/112726360-95012280-8f4f-11eb-9433-d084795c232e.png)
Why would the loss decrease then increase?

**Comment by lissyx** [comment]
Please use discourse for support as documented.
